_episodes/08-putting-it-all-together.md

```diff
@@ -146,25 +146,27 @@ styles and the source codes that create them.
 
 ### `plt` pyplot versus object-based matplotlib
 
-Matplotlib integrates nicely with the Numpy package and can use Numpy arrays
+Matplotlib integrates nicely with the numpy package and can use numpy arrays
```

Comment:
I would suggest changing `Numpy` to `NumPy` rather than its lowercase version `numpy`.